get-started/mining/index.md

```diff
@@ -9,73 +9,72 @@ permalink: /get-started/mining/index.html
-                            <h2>{% t mining.pools %}</h2>
-                        </div>
-                    </div>
-                    <div class="row center-xs">
-                       <p>{% t mining.pools_para1 %} <a href="https://miningpoolstats.stream/monero"> {% t mining.pools_para2 %}</a></p>
-                    </div>
-                </div>
-            </div>
-            <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
-                <div class="info-block">
-                    <div class="row center-xs">
-                        <div class="col">
-                            <h2>{% t mining.benchmarking %}</h2>
-                        </div>
-                    </div>
-                    <div class="row center-xs">
-                       <p><a href="http://monerobenchmarks.info/">{% t mining.benchmarking_para1 %}</a> {% t mining.benchmarking_para2 %}</p>
-                    </div>
+            <div class="full col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                <div class="info-block text-adapt">
+                    <h2>{% t mining.hardware %}</h2>
+                        <p>{% t mining.hardwarep %}</p>
+                    <h2>{% t mining.software %}</h2>
+                        <p>{% t mining.softwarep %}
+                        <ul>
+                            <li><a href="https://github.com/xmrig/xmrig">XMRig</a></li>
+                            <li><a href="https://cryptonote.social/tools/csminer">CSminer</a></li>
+                        </ul>
+                        {% t mining.software_para %}
+                        </p>
                 </div>
             </div>
         </div>
-    </section>
-    <section class="container">
-        <div class="row">      
-            <!-- full block-->
+        <div class="row">
             <div class="full col-lg-12 col-md-12 col-sm-12 col-xs-12">
-                <div class="info-block">
-                    <div class="row center-xs">
-                        <div class="col">
-                            <h2>{% t mining.software %}</h2>
-                        </div>
-                    </div>
-                    <div class="row center-xs">
-                       <p>{% t mining.software_para %}</p>
-                    </div>
-                    <div class="row center-xs">
-                        <p><a href="https://github.com/xmrig/xmrig" target="_blank" rel="noreferrer noopener">XMRig</a></p>
-                    </div>
-                    <div class="row center-xs">
-                        <p><a href="https://github.com/xmrig/xmrig-cuda" target="_blank" rel="noreferrer noopener">XMRig NVIDIA CUDA plugin</a></p>
-                    </div>
+                <div class="info-block text-adapt">
+                    <h2>{% t mining.support %}</h2>
+                        <p>{% t mining.supportp %}</p>
                 </div>
             </div>
-            <!-- end full block -->
         </div>
     </section>
 </div>
+
```

Comment:
nit: just curious, why the extra blank lines?

Reply:
It's common practice to leave one blank line at the end of a document (in fact, Github warns you if the new line is missing). Two are two many tho, i'll fix.